framework/doc/content/source/vectorpostprocessors/PiecewiseFunctionTabulate.md

```diff
@@ -0,0 +1,14 @@
+# PiecewiseFunctionTabulate
+
+!syntax description /VectorPostprocessors/PiecewiseFunctionTabulate
+
+This object can be used to check the function generated by a
```

Comment:
Need a better description here. While it can be used to look at the values from CoarsenedPiecewiseLinear, it can be used to tabulate any Piecewise function.